core/workspace_svg.js

```diff
@@ -537,53 +544,42 @@ Blockly.WorkspaceSvg.prototype.render = function() {
 };
 
 /**
- * Turn the visual trace functionality on or off.
- * @param {boolean} armed True if the trace should be on.
+ * Was used back when block highlighting (for execution) and block selection
```

Comment:
Do we know if anyone else is using this?  If so, the console.warn should probably also say to use highlight directly to highlight multiple blocks during execution.